src/UnitTest/AuthorsTest.cs

```diff
@@ -18,6 +25,21 @@ public class AuthorsTest
             nameof(IAmAFrameworkForXamarin)
         };
 
+        readonly ITestOutputHelper _output;
+
+        Policy _policy = Policy.Handle<WebException>(
+            ex => !ex.Message.Contains("Could not create SSL/TLS secure channel"))
```

Comment:
If you could, you probably would.. But isn't there a better way to detect this than by the means of the exception message?

Reply:
Hmm there probably is. Works OK for now. Will create an issue to improve on this.